hack/lib/util.sh

```diff
@@ -512,25 +512,22 @@ kube::util::ensure_clean_working_dir() {
   done 1>&2
 }
 
-# Ensure that the given godep version is installed and in the path
+# Ensure that the given godep version is installed and in the path.  Almost
+# nobody should use any version but the default.
 kube::util::ensure_godep_version() {
-  GODEP_VERSION=${1:-"v79"}
+  GODEP_VERSION=${1:-"v79"} # this version is known to work
+
   if [[ "$(godep version 2>/dev/null)" == *"godep ${GODEP_VERSION}"* ]]; then
     return
   fi
 
-  kube::util::ensure-temp-dir
-  mkdir -p "${KUBE_TEMP}/go/src"
-
-  GOPATH="${KUBE_TEMP}/go" go get -d -u github.com/tools/godep
-  pushd "${KUBE_TEMP}/go/src/github.com/tools/godep" >/dev/null
-    git checkout -q "${GODEP_VERSION}"
-    GOPATH="${KUBE_TEMP}/go" go install .
-  popd >/dev/null
+  kube::log::status "Installing godep version ${GODEP_VERSION}"
+  go install ./vendor/github.com/tools/godep/
```

Comment:
I like this solution :)

Reply:
there are others that can do the same - gazelle and kazel popped up.  Later.